cmd/build_sensorbee/main.go

```diff
@@ -104,8 +108,12 @@ func loadConfig(path string) (*Config, error) {
 	}
 	// TODO: validation
 
-	config.SubCommands = []string{"run", "shell", "topology", "exp", "runfile"}
-	// TODO: sub commands should be configurable
+	if len(config.SubCommands) == 0 {
+		config.SubCommands = map[string]commandDetail{}
+		for _, sub := range sensorBeeDefaultCommands {
```

Comment:
I think everybody knows that this file is `cmd/build_sensorbee/main.go`, so `defaultCommands` should be ok.